Doc/library/sqlite3.rst

```diff
@@ -337,17 +337,23 @@ Connection Objects
       :meth:`~Cursor.executescript` method with the given *sql_script*, and
       returns the cursor.
 
-   .. method:: create_function(name, num_params, func)
+   .. method:: create_function(name, num_params, func, deterministic=False)
```

Comment:
Yep, I made the same proposal :-) At the C level, it means using "$" instead of "|" before the parameter. Parameters after "$" are keyword-only parameters.